lib/iris/cube.py

```diff
@@ -1981,6 +2140,20 @@ def vector_summary(vector_coords, cube_header, max_line_offset,
                                                                 char=char)
                             vector_summary[index] += line
                     vectors = vectors + cell_measures
+                if ancillary_variables:
+                    # Generate full textual summary for each vector ancillary
+                    # variable - WITH dimension markers.
+                    for index, av in enumerate(ancillary_variables):
+                        dims = self.ancillary_variable_dims(av)
+
+                        for dim in range(len(self.shape)):
+                            width = alignment[dim] - len(vector_summary[index])
+                            char = 'x'
```

Comment:
This line is incorrect. It should be 
```suggestion
                            char = 'x' if dim in dims else '-'
```